### PR TITLE
Fauxdio Group

### PR DESCRIPTION
**Pull request by igreulich**
Remember how much better your life got when we released Fauxdio's? Now you can group them!

### CODE REVIEW

**Comment by mfunkie** [comment]
How would I pass 'inline' to the Fauxdios?

**Comment by igreulich** [comment]
That's a good question...

File: src/Components/FauxdioGroup.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    const renderList = (item) => {
+      return (
+        <Fauxdio
+          radioName={this.props.groupName}
+          radioID={item.id.toString()}
+          value={item.value}
+          label={item.label}
+          key={item.id}
+          checked={item.value === this.props.selectedValue}
+          onChange={this.props.onClick.bind(null, item.value)}
```
Comment (mfunkie) [inline src/Components/FauxdioGroup.jsx]: Will binding to null for `this` stomp any auto-binding on the `onClick` passed?  Could we just do `onChange={() => this.props.onClick(item.value)}` ?

File: src/Components/FauxdioGroup.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    selectedValue: React.PropTypes.oneOfType([
+      React.PropTypes.string,
+      React.PropTypes.number
+    ]),
+    /**
+     * A function to run when an item in the list is clicked. The value of the item is passed to the function
+     */
+    onClick: React.PropTypes.func
+  },
```
Comment (mfunkie) [inline src/Components/FauxdioGroup.jsx]: To solve the inline stuff we could maybe do a common prop object?  Something like

```commonProps: React.PropTypes.shape({ ...Fauxdio.propTypes })```
...
```
<Fauxdio
  {...this.props.commonProps}
  radioName={this.props.groupName}
  radioID={item.id.toString()}
  value={item.value}
  label={item.label}
  key={item.id}
  checked={item.value === this.props.selectedValue}
  onChange={this.props.onClick.bind(null, item.value)} />
```

Naming might need some work, just thinking aloud.

Comment (igreulich) [inline src/Components/FauxdioGroup.jsx]: That's exactly what I was thinking.

File: src/Components/FauxdioGroup.jsx (hunk 1 above)
Comment (igreulich) [inline src/Components/FauxdioGroup.jsx]: I will look at that. It's been a minute, but I think I tried that, and it didn't play nice.

**Comment by mfunkie** [comment]
LGTM

**Comment by mfunkie** [comment]
Still LGTM

**Comment by frockenstein** [comment]
http://all-len-all.com/wp-content/uploads/2015/04/LGBT-sign.jpg

**Pull request merged**